bucket/vieb.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "4.1.0",
+    "description": "Vieb is the Vim Inspired Electron Browser",
+    "homepage": "https://vieb.dev/",
+    "license": "GPL-3.0-or-later",
+    "hash": "1c958e4cba47248213f4b9ede909ea6f57cf24967e953fceb489b5342b1f6f88",
+    "url": "https://github.com/Jelmerro/Vieb/releases/download/4.1.0/Vieb-4.1.0-win.zip",
```

Comment:
What does portable means? Where are application data and configurations stored?

![image](https://user-images.githubusercontent.com/13260377/110093427-ff80d180-7d9a-11eb-80c9-7d77c8fed374.png)

Reply:
The initial configuration is not needed. Vieb reads `%APPDATA%\Vieb\viebrc` file if its created.